.github/spawn-dailies.sh

```diff
@@ -4,6 +4,7 @@ latest_master_url="https://download.nextcloud.com/server/daily/latest-master.tar
 latest_stable26_url="https://download.nextcloud.com/server/daily/latest-stable26.tar.bz2"
```

Comment:
Remove this as well.

Reply:
Oh thanks, good catch. Just forgot about this line. 😄